new.go

```diff
@@ -168,7 +168,7 @@ func resolveImage(ctx context.Context, systemContext *types.SystemContext, store
 		if options.PullPolicy == PullAlways {
 			pulledImg, pulledReference, err := pullAndFindImage(ctx, store, image, options, systemContext)
 			if err != nil {
-				logrus.Debugf("error pulling and reading image %q: %v", image, err)
+				logrus.Debugf("unable to pull and read image %q: %v", image, err)
```

Comment:
@rhatdan, just wanted to note that the errors that are now thrown in the new code in pull.go and util.go below are caught here and thrown out to debug only, not shown to the user directly.  I tried sending it out to the user directly, but that error gets generated for each registry in the list in registries.conf until it is resolved and/or fails looking for the image on localhost.  I thought in the case where it failed looking in a registry and then worked in a second was very misleading so I left the debug here rather than showing the error all the time.